site-src/geps/gep-1364.md

```diff
@@ -0,0 +1,328 @@
+# GEP-1364: Status and Conditions Update
+
+* Issue: [#1364](https://github.com/kubernetes-sigs/gateway-api/issues/1364)
+* Status: Implementable
+
+(See definitions in [Kubernetes KEP][kep-status].
+
+
+
+## TLDR
+
+The status, particularly the Conditions, across the whole Gateway API has very much grown
+grown organically, and so has many inconsistencies and odd behaviors.
+This GEP covers doing a review and consolidation to make Condition behavior consistent
+across the whole API.
+
+## Goals
+
+* Update Conditions design to be consistent across Gateway API resources
+* Provide a model and guidelines for Conditions for future new resources
+* Specify changes to conformance required for Condition updates
+
+## Non-Goals
+
+
+
+## Introduction
+
+Gateway API currently has a lot of issues related to status, especially that
+status is inconsistent ([#1111][1111]), that names are hard to understand ([#1110][1110]),
+Reasons aren't explained properly ([#1362][1362]), and so on.
+
+As the API has grown, the way we talk about resources has changed a lot, and some of the
+status design hasn't been updated since resources were created.
+
+So, for example,  we have GatewayClass with `Accepted`, Gateway with `Scheduled`,
+the Gateway Listeners with `Detached` (which you want to be `false`, unlike the previous
+two), and then Gateways and Gateway Listeners have `Ready`, but Route doesn't (and which
+also you want to be `true`).
+
+This document lays out large-scale changes to the way that we talk about resources,
+and the Conditions to match them. This means that there will be an unavoidable break
+in what constitutes a healthy or unhealthy resource, and there will be changes
+required for all conformant implementations to stay conformant.
```

Comment:
Maybe specifically mention the concept of being conformant with a specific release of the API?
```suggestion
required for all implementations to be conformant with the release that includes these changes.
```